toqito/state_props/entanglement_of_formation.py

```diff
@@ -65,7 +66,7 @@ def entanglement_of_formation(rho: np.ndarray, dim: list[int] | int = None) -> f
 
     # User can specify dimension as integer.
     if isinstance(dim, int):
-        dim = np.array([dim, max(dim_x, dim_y) / dim], dtype=int)
+        dim = np.array([dim, max(dim_x, dim_y) / dim], dtype=int) # pylint: disable=redefined-variable-type
```

Comment:
@vprusso Would it be better to ignore errors related to redefined variables? 

Here, the error raised was that the variable was re-defined from `int` to `array`.